tembo-cli/src/cmd/apply.rs

```diff
@@ -748,31 +754,36 @@ pub fn get_rendered_dockerfile(
     Ok(rendered_dockerfile)
 }
 
-fn get_postgres_config(instance_settings: &InstanceSettings) -> String {
+fn get_postgres_config(
+    instance_ps_config: Option<HashMap<String, Value>>,
+    postgres_configs: &std::option::Option<Vec<ControllerPgConfig>>,
+) -> String {
     let mut postgres_config = String::from("");
     let qoute_new_line = "\'\n";
     let equal_to_qoute = " = \'";
-    let stack_type = ControllerStackType::from_str(instance_settings.stack_type.as_str())
-        .unwrap_or(ControllerStackType::Standard);
-
-    let stack = get_stack(stack_type);
-
-    if stack.postgres_config.is_some() {
-        for config in stack.postgres_config.unwrap().iter() {
-            postgres_config.push_str(config.name.as_str());
-            postgres_config.push_str(equal_to_qoute);
-            postgres_config.push_str(format!("{}", &config.value).as_str());
-            postgres_config.push_str(qoute_new_line);
+    if postgres_configs.is_some() {
+        for p_config in postgres_configs.clone().unwrap().into_iter() {
+            match p_config.value {
+                ControllerConfigValue::Single(val) => {
+                    postgres_config.push_str(p_config.name.as_str());
+                    postgres_config.push_str(equal_to_qoute);
+                    postgres_config.push_str(&val);
+                    postgres_config.push_str(qoute_new_line);
+                }
+                ControllerConfigValue::Multiple(vals) => {
+                    for val in vals {
+                        postgres_config.push_str(p_config.name.as_str());
+                        postgres_config.push_str(equal_to_qoute);
+                        postgres_config.push_str(val.as_str());
+                        postgres_config.push_str(qoute_new_line);
+                    }
```

Comment:
Would this section result in configs like this?

```
shared_preload_libraries=pg_cron
shared_preload_libraries=vectorize
shared_preload_libraries=pg_partman_bgw
```

Reply:
Yes for now. I'm going to work on the `shared_preload_libraries` next.

Reply:
There is a [to_postgres](https://github.com/tembo-io/tembo/blob/51154b140c5963981c8070fb606347c809a20492/tembo-operator/src/apis/postgres_parameters.rs#L136-L141)() method on the ConfigValue struct

Example converting the [enum to string](https://github.com/tembo-io/tembo/blob/51154b140c5963981c8070fb606347c809a20492/tembo-operator/src/apis/postgres_parameters.rs#L421-L424)

Reply:
Shouldn't it be something like the following?

```
shared_preload_libraries='pg_cron, vectorize, pg_partman_bgw'
```

Reply:
yes @vrmiguel , the `to_postgres()` method does exactly that

Reply:
Addressed it for now... But will require more work to handle `loadable_libraries`, `priority` etc